src/ProjectConfiguration.ts

```diff
@@ -104,6 +106,14 @@ const permissionTypes = [
     description: 'Application may track an exercise.',
     sdkVersion: '>=3.0.0',
   },
+  {
```

Comment:
TODO: we should write tests here to make sure all the AppProjectConfiguration capabilities already (e.g. access_app_cluster_storage)  have unique ids and maybe other requirements (e.g. sdkVersion is semver, etc.), but not in this PR (if we don't have test already)

Reply:
Sounds like a good thing to file a bug for. As a little history, this came across from the Studio codebase and we haven't changed it much since. It's tech debt we'll need to repay at some point by re-structuring it (and de-duplicating it with Studio).

Reply:
I fixed the unique IDs part partially by converting permission keys to a string enum. Still could use a ticket to come back and refactor + add tests though.